catalog/ui/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.view.js

```diff
@@ -54,7 +55,10 @@ define([
             this.model.collection.remove(this.model);
         },
         handleDeleted: function(){
-            this.model.startSearch(true);
+            this.model.startSearch({
+                limitDeleted: true,
```

Comment:
You can remove the false part of the object since that's now set by default. 
(similar on the historic function below)